packages/simple-hub/src/wallet/deposit.ts

```diff
@@ -27,17 +27,24 @@ export function depositsToMake(
 ): {channelId: string; amountToDeposit: BigNumber}[] {
   const simpleAllocationStates = _.filter(isSimpleAllocationState, message.signedStates);
   return simpleAllocationStates
-    .filter(state => state.participants.length === 2)
```

Comment:
No need for a closure and return syntax as far as I can tell

Reply:
Ah, these changes were made as it seems finicky to add breakpoints without a closure + return statement.